PR: https://github.com/odoo/odoo/pull/85199

From: e655e8da8a17e766021bb4787648939118682ad3
From: Julien Castiaux
Date: 2022-02-23 12:59:52

Structural Changes: 6
Total Changes: 6

Revert "[IMP] base: add prefetching group feature"

This reverts commit 041fe5e21e625e6ea80d26ecd041491c89cbba12.

Part-of: odoo/odoo#85199

================================= pseudo patch: =================================

--- a/addons/website/models/mixins.py
+++ b/addons/website/models/mixins.py
@@ -21,9 +21,9 @@ class SeoMetadata(models.AbstractModel):
     _description = 'SEO metadata'
 
     is_seo_optimized = fields.Boolean("SEO optimized", compute='_compute_is_seo_optimized')
-    website_meta_title = fields.Char("Website meta title", translate=True, prefetch="website_meta")
-    website_meta_description = fields.Text("Website meta description", translate=True, prefetch="website_meta")
-    website_meta_keywords = fields.Char("Website meta keywords", translate=True, prefetch="website_meta")
+    website_meta_title = fields.Char("Website meta title", translate=True)
+    website_meta_description = fields.Text("Website meta description", translate=True)
+    website_meta_keywords = fields.Char("Website meta keywords", translate=True)
     website_meta_og_img = fields.Char("Website opengraph image")
     seo_name = fields.Char("Seo name", translate=True, prefetch=True)
 
